frontend/packages/core/src/Input/toggle-button-group.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  ToggleButton as MuiToggleButton,
+  ToggleButtonGroup as MuiToggleButtonGroup,
+} from "@material-ui/lab";
+
+export interface ToggleButtonGroupProps {
```

Comment:
Should we expose the ToggleButtonGroup disabled and orientation props as well?

Reply:
Oh I was looking at V5 docs, we can not include that for now since it doesn't exist yet.